src/RosAria.cpp

```diff
@@ -492,6 +522,20 @@ void RosAriaNode::publish()
   position.header.frame_id = frame_id_odom;
   position.child_frame_id = frame_id_base_link;
   position.header.stamp = ros::Time::now();
+
+  if(position.twist.twist.linear.x == 0 && 
+    position.twist.twist.linear.y == 0 && 
+    position.twist.twist.angular.z ==0 )
```

Comment:
Minor spacing issue, otherwise should these be checking for some epsilon as opposed to zero?